src/Resources/config/actions.php

```diff
@@ -73,6 +73,7 @@
                 new ReferenceConfigurator('validator'),
                 new ReferenceConfigurator('sonata.admin.form.data_transformer_resolver'),
                 new ReferenceConfigurator('property_accessor'),
+                new ReferenceConfigurator('sonata.render_element.twig.runtime'),
```

Comment:
```suggestion
                new ReferenceConfigurator('sonata.admin.render_element.twig.runtime'),
```

To follow other services name. And replace other names too.

Reply:
I named everything  `sonata.admin.twig.foo_extension/runtime` and added deprecated alias.